orderedmap_test.go

```diff
@@ -261,16 +260,6 @@ func assertLenEqual(t *testing.T, om *OrderedMap, expectedLen int) {
 	assert.Equal(t, expectedLen, om.list.Len())
 }
 
-func shuffle(a []interface{}) {
```

Comment:
Mmh good catch, the `TestShuffle` actually doesn't use this, dunno what I was thinking when I wrote it :/ I'll fix that later, thanks for noticing!